### PR TITLE
Publish base62 module

### PR DESCRIPTION
**Pull request by yungsters**
Makes `base62` available in the `fbjs` package.

### CODE REVIEW

File: src/crypto/__tests__/base62-test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+relay
```
Comment (zpao) [inline src/crypto/__tests__/base62-test.js]: relay eh?

**Comment by zpao** [comment]
👍. update the changelog to add for this and just call it v0.5.1

File: src/crypto/__tests__/base62-test.js (hunk 1 above)
Comment (yungsters) [inline src/crypto/__tests__/base62-test.js]: Haha, it's the same for `crc32`.

**Comment by yungsters** [comment]
Thanks!

**Pull request merged**